server/utils/match/index.ts

```diff
@@ -195,6 +190,37 @@ export const occupancyViewSummaryListForMatchingDetails = (
   ]
 }
 
+export const textRow = (label: string, value: string): SummaryListItem => ({
```

Comment:
These utils look like they could be used widely. there is a similar fn in `utils/placements/index.ts` for instance.
Perhaps it would be good to put these somewhere global - `utils/formUtils` say.

Reply:
I've refactored the above a bit but not gone on to replace all instances -- this would seem like a sizeable change to add to this. Thoughts?